MonitoringKit/Monitoring_X8/src/Monitoring_X8.cpp

```diff
@@ -56,7 +56,7 @@ void newDataCallback(const bme68xData data, const bsecOutputs outputs, Bsec2 bse
 // Wifi & MQTT
 WiFiClientSecure wifiClient;
 PubSubClient client(wifiClient);
-const bool useEnterprise = 0;
+const bool useEnterprise = 1;
```

Comment:
Should probably move this into secrets eventually... doesn't really matter for this PR